net/netcheck/netcheck.go

```diff
@@ -52,9 +52,9 @@ var (
 
 // The various default timeouts for things.
 const (
-	// overallProbeTimeout is the maximum amount of time netcheck will
+	// ReportTimeout is the maximum amount of time netcheck will
 	// spend gathering a single report.
-	overallProbeTimeout = 5 * time.Second
+	ReportTimeout = 5 * time.Second
 	// stunTimeout is the maximum amount of time netcheck will spend
```

Comment:
do we need any new tests ensuring certain sums of constants are less than other constant(s)?

Reply:
I added a test in https://github.com/tailscale/tailscale/pull/13395/commits/794bfa70b5e4b60225b6824dbd318b2a852872d1, but this doesn't involve any summing. I plan to follow this PR with the larger happy eyeballs / probePlan change, where those tests should naturally fall out.